### PR TITLE
Patch fixes

### PR DESCRIPTION
**Pull request by cloudhead**
*See commits.*

### CODE REVIEW

File: radicle-cli/src/commands/patch/checkout.rs [hunk 1]
```diff
@@ -1,38 +1,77 @@
...
+            );
+            let url = git::Url::from(stored.id).with_namespace(author);
+
+            // Fetch only the ref pointing to the patch revision.
+            working.remote_anonymous(url.to_string().as_str())?.fetch(
+                &[&format!("{refstr}:{remote_branch}")],
+                None,
+                None,
+            )?;
+            working.find_commit(patch_head.into())?
```
Comment (slack-coder) [inline radicle-cli/src/commands/patch/checkout.rs:58]: Maybe move this code into a private method?  That way the condition and resulting actions are clear.

**Comment by slack-coder** [comment]
Otherwise :+1:

File: radicle-cli/src/commands/patch/checkout.rs [hunk 2]
```diff
@@ -1,38 +1,85 @@
...
+use radicle::git::RefString;
 use radicle::storage::git::Repository;
+use radicle::storage::ReadRepository;
+
+use crate::terminal as term;
 
 pub fn run(
-    storage: &Repository,
-    git_workdir: &git::raw::Repository,
+    stored: &Repository,
```
Comment (FintanH) [inline radicle-cli/src/commands/patch/checkout.rs:13]: Why the rename, out of interest?

Comment (cloudhead) [reply radicle-cli/src/commands/patch/checkout.rs:13]: `storage` is confusing when it doesn't refer to `Storage`, no?

I think a nice convention if there are two repositories is `stored` for the stored copy and `working` for the working copy. That was the idea.

**Pull request merged**